python/adelphi/bin/adelphi

```diff
@@ -54,7 +83,7 @@ log = logging.getLogger('adelphi')
 @click.option('--anonymize/--no-anonymize', default=True, show_default=True, help="Enable/disable schema anonymization")
 @click.option('--output-dir', help='Directory schema files should be written to. If not specified, it will write to stdout')
 @click.option('--purpose', help='Comments on the anticipated purpose of this schema.  Optional.')
-@click.option('--maturity', help='The maturity of this schema.  Sample values would include ‘alpha’, ‘beta’, ‘dev’, ‘test’ or ‘prod’.  Optional.')
+@click.option('--maturity', help="The maturity of this schema.  Sample values would include 'alpha', 'beta', 'dev', 'test' or 'prod'.  Optional.")
```

Comment:
The backticks were causing a problem for Python 2.x; it wasn't happy with the non-ASCII-ness of these characters.